charts/datadog/templates/agent-priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if or .Values.providers.gke.autopilot .Values.datadog.priorityClassName }}
+apiVersion: scheduling.k8s.io/v1
+description: Used for Datadog Agent Components to be scheduled with higher priority.
+kind: PriorityClass
+metadata:
+  name: {{ default "datadog-agent-priority-class" .Values.agents.priorityClassName }}
```

Comment:
The name should contains the release name, else it will not work with 2 charts deployment in the same namespace

Reply:
good point

Reply:
maybe you can create an help for the name. since it will be used in 2 places